game/debuffHeal.cc

```diff
@@ -15,3 +17,9 @@ void DebuffHeal::useItemOverride(Hackmon &target) const {
     target.debuff = Debuff{HEALTHY};
   }
 }
+
+void DebuffHeal::printItem() {
+  cout << left << "Item: " << setw(16) << "Debuff Heal";
+  cout << "Debuff: " << setw(12) << debuffString[debuff];
+  if (numberBattling > 1) cout << "Scope: " << setw(18) << (scope==0 ? "Single" : "All");
```

Comment:
Use the actual enum value instead of the associated int, just to be more readable.
```suggestion
  if (numberBattling > 1) cout << "Scope: " << setw(18) << (scope==SINGLE ? "Single" : "All");
```